Makefile

```diff
@@ -117,118 +86,47 @@ update_devworkspace_api:
 _init_devworkspace_crds:
 	./update_devworkspace_crds.sh --init --api-version $(DEVWORKSPACE_API_VERSION)
 
-### update_devworkspace_crds: pull latest devworkspace CRDs to ./devworkspace-crds. Note: pulls master branch
+### update_devworkspace_crds: Pulls the latest devworkspace CRDs to ./devworkspace-crds. Note: pulls master branch
 update_devworkspace_crds:
 	./update_devworkspace_crds.sh --api-version $(DEVWORKSPACE_API_VERSION)
 ###### End rules for dealing with devfile/api
 
-### test: Run tests
+### test: Runs tests
 ENVTEST_ASSETS_DIR = $(shell pwd)/bin/testbin
 test: generate fmt vet manifests
 	mkdir -p $(ENVTEST_ASSETS_DIR)
 	test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
 	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test $(shell go list ./... | grep -v test/e2e) -coverprofile cover.out
 
-### test_e2e: runs e2e test on the cluster set in context. DevWorkspace Operator must be already deployed
+### test_e2e: Runs e2e test on the cluster set in context. DevWorkspace Operator must be already deployed
 test_e2e:
 	CGO_ENABLED=0 go test -v -c -o bin/devworkspace-controller-e2e ./test/e2e/cmd/workspaces_test.go
 	./bin/devworkspace-controller-e2e -ginkgo.failFast
 
-### test_e2e_debug: runs e2e test in debug mode, so it's possible to connect to execution via remote debugger
+### test_e2e_debug: Runs e2e test in debug mode, so it's possible to connect to execution via remote debugger
 test_e2e_debug:
 	dlv test --listen=:2345 --headless=true --api-version=2 ./test/e2e/cmd/workspaces_test.go -- --ginkgo.failFast
 
-### manager: Build manager binary
+### manager: Builds the manager binary
 manager: generate fmt vet
 	go build -o bin/manager main.go
```

Comment:
While I'm nitpicking, we should just remove this target from the file completely. It's vestigial at this point.

Reply:
Okay, sounds good, I've removed it!